docs/en/query_language/functions/date_time_functions.md

```diff
@@ -169,6 +169,72 @@ Converts a date or date with time to a UInt16 number containing the ISO Year num
 
 Converts a date or date with time to a UInt8 number containing the ISO Week number.
 
+## week(date[,mode])
+This function returns the week number for date or datetime. The two-argument form of week() enables you to specify whether the week starts on Sunday or Monday and whether the return value should be in the range from 0 to 53 or from 1 to 53. If the mode argument is omitted, the default mode is 0.
+`toISOWeek() `is a compatibility function that is equivalent to `week(date,3)`.
+The following table describes how the mode argument works.
+
+| Mode | First day of week | Range |  Week 1 is the first week … |
+| ----------- | -------- | -------- | ------------------ |
+|0|Sunday|0-53|with a Sunday in this year
+|1|Monday|0-53|with 4 or more days this year
+|2|Sunday|1-53|with a Sunday in this year
+|3|Monday|1-53|with 4 or more days this year
+|4|Sunday|0-53|with 4 or more days this year
+|5|Monday|0-53|with a Monday in this year
+|6|Sunday|1-53|with 4 or more days this year
+|7|Monday|1-53|with a Monday in this year
+|8|Sunday|1-53|contains January 1
+|9|Monday|1-53|contains January 1
```

Comment:
I didn't see the modes `8` and `9` in the [MySQL documentation](https://dev.mysql.com/doc/refman/5.5/en/date-and-time-functions.html#function_week). Did you take them from somewhere else?

Reply:
FYI:
https://stackoverflow.com/questions/55658435/python-north-american-work-week-number-from-datetime
https://www.calendar-12.com/week_number

Reply:
thanks